components/SingleSelectRadio/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import { ComponentContainer, RadioInput, RadioLabel } from './styles';
+
+interface Option {
+  label: string;
+  value: string;
+}
+
+interface SingleSelectRadioGroupProps {
+  name: string;
+  options: Option[];
+  defaultValue?: string;
+  onChange?: (value: string) => void;
+}
+
+export default function SingleSelectRadioGroup({
```

Comment:
Maybe we can rename this to `RadioGroup` (and the file to `components/RadioGroup/index.tsx`) ?

Reply:
you also might consider typing this component like this 

export default function SingleSelectRadioGroup<T>

and Option as Option<T> in case the values are not strings, but boolean, for example. this might come up in your next sprint where you'll use the RadioComponent in Onboarding